frontend/packages/core/src/Input/text-field.tsx

```diff
@@ -113,9 +117,9 @@ const StyledTextField = styled(BaseTextField)({
     marginTop: "7px",
     lineHeight: "16px",
     marginLeft: "0px",
-
+    color: `${TEXT_FIELD_COLOR_MAP[props.color] || TEXT_FIELD_COLOR_MAP.default}`,
```

Comment:
Not relevant to this line but we also need to update the placeholder color to pull from the map now